### PR TITLE
Improve description about return value types of handler method in reference doc

### PR DESCRIPTION
**Pull request by kazuki43zoo**
Improve as follows:
- Add an WebAsyncTask
- Add a CompletionStage
- Trivial improvement

Please review.
I have signed and agree to the terms of the SpringSource Individual Contributor License Agreement.

### CODE REVIEW

File: src/asciidoc/web-mvc.adoc [hunk 1]
```diff
@@ -1257,15 +1257,20 @@ The following are the supported return types:
 * An `HttpHeaders` object to return a response with no body.
 * A `Callable<?>` can be returned when the application wants to produce the return value
   asynchronously in a thread managed by Spring MVC.
+* An `WebAsyncTask<?>` can be returned when the application wants to produce the return
```
Comment (sbrannen) [inline src/asciidoc/web-mvc.adoc]: That should be "A `WebAsyncTask<?>`", not "An".

File: src/asciidoc/web-mvc.adoc [hunk 2]
```diff
@@ -1257,15 +1257,20 @@ The following are the supported return types:
...
+  value from a thread of its own choosing.
 * A `DeferredResult<?>` can be returned when the application wants to produce the return
   value from a thread of its own choosing.
 * A `ListenableFuture<?>` can be returned when the application wants to produce the return
   value from a thread of its own choosing.
+* A class which implements a `CompletionStage<?>` interface (e.g. `CompletableFuture<?>`)
```
Comment (sbrannen) [inline src/asciidoc/web-mvc.adoc]: Change to:

A class which implements `CompletionStage<?>` (e.g., `CompletableFuture<?>`)

**Comment by kazuki43zoo** [comment]
@sbrannen , thanks for review comments.
I've reflected review comments.

**Comment by sbrannen** [comment]
I apologize for the belated reply: this PR slipped under our radar.

In the interim similar changes have already been made to the documentation, so I will close this PR.

**Pull request closed without merging**